src/js/angular/import/directives/import-resource-tree.directive.js

```diff
@@ -22,30 +30,93 @@ function importResourceTreeDirective() {
             // Public variables
             // =========================
             $scope.displayResources = [];
+            $scope.TYPE_FILTER_OPTIONS = TYPE_FILTER_OPTIONS;
+            $scope.filterByType = TYPE_FILTER_OPTIONS.ALL;
+            $scope.filterByFileName = '';
 
             // =========================
             // Private variables
             // =========================
             let selectedResources = [];
 
+            // =========================
+            // Public functions
+            // =========================
+            $scope.selectionChanged = (($event, resource) => {
```

Comment:
If you don't need the event, then remove the parameter

Reply:
Removed the event